Automattic/wp-calypso
# SALP: Add interface toggle to enable/disable sites dashboard as landing page

#### Proposed Changes

* Add a component that toggles user's preference for setting sites dashboard as landing page.

#### Testing Instructions

- Go to `/me/account`
- Verify sites as landing page toggle is shown when  config `'sites-as-landing-page': true` is set
- Verify sites as landing page toggle is not shown when  config `'sites-as-landing-page': false` 
- Verify event `calypso_settings_sites_dashboard_landing_page_toggle ` event is sent in console when toggle on/off
- Run `localStorage.setItem( 'debug', 'calypso:analytics*' );` in console to see events.


* 
![image](https://user-images.githubusercontent.com/47489215/203648986-6c735db2-3a52-4498-825c-e523d26f9d09.png)


#### Pre-merge Checklist





Related to #
